examples/Ethereum Crypto Trend Analysis Notebook.ipynb

```diff
@@ -0,0 +1,487 @@
+          ]
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "application/vnd.colab-display-data+json": {
+              "pip_warning": {
+                "packages": [
+                  "defusedxml"
+                ]
+              },
+              "id": "60e809cfefe84b50ab434d4bb1885607"
+            }
+          },
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from openbb import obb\n",
+        "import pandas as pd\n",
+        "import matplotlib.pyplot as plt\n",
+        "\n",
+        "print(\"Data fetched successfully!\")  # More specific than \"No errors!\"\n",
+        "\n",
+        "# Download historical Ethereum (ETH) to USD price data for a specific period.\n",
+        "btc_data = obb.crypto.price.historical(symbol='ETH-USD', interval='1d', start_date='2024-02-01', end_date='2024-06-01')\n",
+        "\n",
+        "# Extract dates and closing prices from the downloaded data\n",
+        "dates = [data.date for data in btc_data.results]\n",
+        "prices = [data.close for data in btc_data.results]\n",
+        "volumes = [data.volume for data in btc_data.results]\n",
+        "\n",
+        "# Visualize the Ethereum price trend\n",
+        "plt.figure(figsize=(12, 6))\n",
+        "plt.plot(dates, prices)\n",
+        "plt.title('Ethereum Price Trend (Feb 1 - Jun 1, 2024)')\n",
+        "plt.xlabel('Date')\n",
+        "plt.ylabel('Price (USD)')\n",
+        "plt.xticks(rotation=45)\n",
+        "plt.tight_layout()\n",
+        "plt.show()\n",
+        "\n",
+        "# Function to calculate Simple Moving Average (SMA) for a given window\n",
+        "def calculate_ma(data, window):\n",
+        "  \"\"\"\n",
+        "  This function calculates the Simple Moving Average (SMA) of a time series data for a specified window size.\n",
+        "  \"\"\"\n",
+        "  return [sum(data[max(i-window+1, 0):i+1])/min(i+1, window) for i in range(len(data))]\n",
```

Comment:
This has a slight issue that will raise a bug if window>len(data).

This notebook in general could be simplified moving to pandas and using the .to_df() method.  This way you can just use rolling